NPE in studies view

Caused by: java.lang.NullPointerException
        at fi.otavanopisto.muikku.plugins.transcriptofrecords.rest.TranscriptofRecordsRESTService.getVops(TranscriptofRecordsRESTService.java:283)

--- a/muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/transcriptofrecords/rest/TranscriptofRecordsRESTService.java
+++ b/muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/transcriptofrecords/rest/TranscriptofRecordsRESTService.java
@@ -320,7 +320,7 @@ public class TranscriptofRecordsRESTService extends PluginRESTService {
       for (GradingScaleItem gradingScaleItem : gradingController.listGradingScaleItems(gradingScale)) {
         GradingScaleItemCoordinates newItemKey = new GradingScaleItemCoordinates(
             gradingScaleIdentifier,
-            new SchoolDataIdentifier(gradingScaleItem.getGradingScaleIdentifier(), gradingScaleItem.getSchoolDataSource()));
+            new SchoolDataIdentifier(gradingScaleItem.getIdentifier(), gradingScaleItem.getSchoolDataSource()));
         gradingScaleCache.put(newItemKey, gradingScaleItem);
       }
     }

